src/pulselistener/tests/test_monitoring.py

```diff
@@ -206,3 +210,38 @@
     assert NotifyMock.email_obj['content'] == content
     assert NotifyMock.email_obj['template'] == 'fullscreen'
     assert monitoring.stats == {}
+
+
+@pytest.mark.asyncio
+async def test_monitoring_retry_exceptions(QueueMock, NotifyMock):
+    monitoring = Monitoring(1)
+    monitoring.emails = ['pinco@pallino']
```

Comment:
Great email!

Reply:
I have no idea who this is (some italian guy ?), i just copied & pasted from above :smile: